relay-server/src/utils/rate_limits.rs

```diff
@@ -242,7 +249,9 @@ impl Enforcement {
                     event_id: envelope.event_id(),
                     remote_addr: envelope.meta().remote_addr(),
                     category: limit.category,
-                    quantity: limit.quantity,
+                    // XXX: on the limiter we have quantity of usize, but in the protocol
+                    // and data store we're limited to u32.
+                    quantity: limit.quantity as u32,
```

Comment:
Since I only changed it on the protocol to a u32 we have a few of these now which are just suboptimal casts.